jetty-plus/src/main/java/org/eclipse/jetty/plus/webapp/EnvConfiguration.java

```diff
@@ -40,7 +40,6 @@
 import org.eclipse.jetty.util.log.Log;
 import org.eclipse.jetty.util.log.Logger;
 import org.eclipse.jetty.webapp.AbstractConfiguration;
-import org.eclipse.jetty.webapp.Configuration;
 import org.eclipse.jetty.webapp.WebAppClassLoader;
```

Comment:
Not a huge fan of making incidental  changes in other files, although as this is just a single line, ok.

Reply:
I was digging into EnvConfiguration to see if it was doing anything strange with exceptions, didn't notice the removal of the unusued import.  I can undo easily enough.

Reply:
Actually, not that easy to undo. :-)
The editor refuses to let me add back an import that isn't used, even on revert!